roxie/ccd/ccdmain.cpp

```diff
@@ -220,7 +220,7 @@ StringBuffer tempDirectory;
 ClientCertificate clientCert;
 bool useHardLink;
 
-unsigned maxFileAge[2] = {0xffffffff, 60*60*1000}; // local files don't expire, remote expire in 1 hour, by default
+unsigned __int64 maxFileAgeNS[2] = {0xffffffffffffffff, 60*60*1000*1000000L}; // local files don't expire, remote expire in 1 hour, by default
```

Comment:
Might both need LL/ULL extensions  long may only be 32 bits.

Reply:
Will change